edvart/report_sections/bivariate_analysis.py

```diff
@@ -980,6 +982,12 @@ def contingency_table(
         annot_kws={"fontsize": fontsize},
         square=True,
     )
+    if size_factor == "auto":
+        n_digits_max = np.floor(np.log10(table.max().max()))
+        size_factor = max(
+            0.72,
+            0.18 * (1 + n_digits_max),
```

Comment:
Previously, the default was `0.7`. Why changing it to `0.72` and why using `0.18` per digit (`+1`)? Are these numbers obtained empirically from testing how to output looks like?

I'd add a short explanatory comment how these numbers were obtained.

Reply:
The numbers were chosen empirically. Added a comment.
The change from `0.7` to `0.72` was to make it match (4 * 0.18 = 0.72), otherwise the `size_factor` would be `0.7` for 3-digit numbers and `0.72` for 4-digit numbers.
I figured it a) does not make a difference due to rounding etc. -> The change has no effect, or b) would make a difference and would be ugly due to *slightly* different cell sizes.

Reply:
Also moved the +1 to the computation of `n_digits_max` since the number of digits is floor(log (n)) + 1.